test/gitea_access_control_test.go

```diff
@@ -241,6 +243,109 @@ func TestGiteaACLCommentsAllowing(t *testing.T) {
 	}
 }
 
+// TestGiteaACLCommentsAllowingRememberOkToTestFalse tests when unauthorized user sends a PR the status shows as pending
+// unless the authorized user adds a comment like /ok-to-test, When authorized user adds those comments
+// the status of CI shows as success. Now non authorized user pushes to PR, the CI will again go to pending
+// and require /ok-to-test again from authorized user.
+func TestGiteaACLCommentsAllowingRememberOkToTestFalse(t *testing.T) {
+	comment := "/ok-to-test"
```

Comment:
Can we make it as reusable variable?
Because i see `/ok-to-test` usage multiple places in this file so

Reply:
done